librz/util/buf.c

```diff
@@ -299,7 +318,19 @@ RZ_API RzBuffer *rz_buf_new_file(const char *file, int perm, int mode) {
 	return new_buffer(RZ_BUFFER_FILE, &u);
 }
 
-RZ_API RzBuffer *rz_buf_new_mmap(const char *filename, int perm, int mode) {
+/**
+ * \brief Creates a new buffer from a file using rz_file_mmap.
+ * \param file The filename used to create the new buffer.
+ * \param perm Same meaning than the symbolic constants defined in sys/stat.h.
+ * \param mode Same meaning than the symbolic constants use with open (fcntl.h).
+ * \return Return the new allocated buffer.
+ *
+ * \see rz_file_mmap()
+ *
+ * The function creates a new allocated buffer using the RZ_BUFFER_MMAP as back end.
+ * The buffer content is synchronized with file content.
```

Comment:
The function creates a new buffer synchronized with the file content, using mmap to access the file.